### DISCUSSION TITLE
Suggestion: Vertical Tabs...

### DISCUSSION THREAD

**Discussion by DrProbie** [microsoft/terminal#16246] [Ideas]
Has anyone given thought to implementing the Vertical Tabs feature...?

I turned it on in Edge way back when it showed up in the Canary build and never switched back. When I opened up the Terminal this morning, I suddenly thought "Now wouldn't that be neat?"

**Comment by j4james** [comment]
Yeah, support for vertical tabs is tracked in #835. The blocking issue right now is that it's dependent on upstream support in the WinUI library (https://github.com/microsoft/microsoft-ui-xaml/issues/2194).

**Reply by zadjii-msft** [reply]
Yep, this is right. Thanks!
